readme.txt

```diff
@@ -231,6 +231,9 @@ Remember to always make a backup of your database and files before updating!
 
 = [6.0.1] TBD =
 
+* Tweak - Ensure the Fast-forward link uses the correct, customizable color as other links. [TEC-3962]
```

Comment:
did we have a reversion?

Reply:
Not sure. What do you mean? I conflict resolved and I think it ended up re-ordering.